### PR TITLE
CU-86897eb76 Add doc for single configuration endpoint

### PR DESCRIPTION
**Pull request by quan-managexr**

### CODE REVIEW

**Comment by quan-managexr** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/ManageXR/public-api-docs/8?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#8** <a href="https://app.graphite.dev/github/pr/ManageXR/public-api-docs/8?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#7** <a href="https://app.graphite.dev/github/pr/ManageXR/public-api-docs/7?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @quan-managexr and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: openapi.yaml [hunk 1]
```diff
@@ -692,7 +692,261 @@ paths:
...
+                                 type: string
+                                 enum:
+                                    - HOME_SCREEN
+                                    - KIOSK
+                                    - DEFAULT
+                                 description: Device experience set on the configuration.
+                              apps:
+                                 type: object
+                                 properties:
+                                    app-id:
```
Comment (quan-managexr) [inline openapi.yaml:746]: This (and similar pattern for other assets) is a workaround to both
1. convey the fact that `apps` is a map that maps an app id to the app object
2. document the structure of each app object

The official OpenAPI syntax for this is 

```
apps:
  type: object
  additionalProperties:
     type: object
     properties:
       // define the structure of an app object here
```

More on `additionalProperties` here: https://swagger.io/docs/specification/data-models/dictionaries/.

However, mintlify can't seem to parse `additionalProperties` field. I've searched up their doc but they didn't mention this.

Comment (taylorlallas) [reply openapi.yaml]: As discussed, we're updating the endpoint to return arrays rather than object maps. Resolving this!

**Review by taylorlallas — APPROVED** [review]

**Comment by taylorlallas** [comment]
### Merge activity

* **Aug 12, 9:15 PM EDT**: @taylorlallas started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/ManageXR/public-api-docs/8).

**Pull request merged**